bot/cogs/wolfram.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from discord.ext import commands
+from urllib import parse
+from io import BytesIO
+import aiohttp
+import os
+
+
+APPID = os.environ.get("WOLFRAM_APPID")
+QUERY = "http://api.wolframalpha.com/v2/{request}?{data}"
+
+class Wolfram(commands.Cog):
+    """command for wolfram search"""
+    def __init__(self, bot):
+        self.bot = bot
+
+    # wolfram command, takes in a search and gives the result
+    # PS, I didn't totally steal it from seasonal bot
+    @commands.command(brief="Takes in a wolfram search and displays the result", usage=".wolfram [query]")
+    async def wolfram(self, ctx, *, query):
+
+        url_str = parse.urlencode({
+            "i": query,
+            "appid": APPID,
+        })
+
+        query_final = QUERY.format(request="simple", data=url_str)
+
+        async with ctx.channel.typing():
+            async with aiohttp.ClientSession() as cs:
+                async with cs.get(query_final) as response:
+                    status = response.status
+                    image_bytes = await response.read()
+
+                    f = discord.File(BytesIO(image_bytes), filename="image.png")
```

Comment:
non descriptive variable name that is used 30 lines later, should have a descriptive name. Consider `image_file`